vignettes/using-board-url.Rmd

```diff
@@ -0,0 +1,208 @@
+```
+
+We confirm that there is a file called `_pins.yaml`:
+
+```{r}
+withr::with_dir(board$path, fs::dir_ls())
+```
+
+We can inspect its contents to see each pin in the board, and each version of each pin:
+
+```yaml
+`r if (rlang::is_installed("webfakes")) paste(readLines(fs::path(board$path, "_pins.yaml")), collapse = "\n")`
+```
+
+At this point, we would publish the folder containing the board as a part of a web site.
+For the purpose of this vignette, we pretend that we have served the folder from our fake website, `https://not.real.website.co/pins/`.
+
+```{r board-serve}
+#| echo: false
+board_server <- webfakes::new_app()
+board_server$use(webfakes::mw_static(root = board$path))
+board_process <- webfakes::new_app_process(board_server)
+
+web_board <- board_url(board_process$url())
+```
+
+## Consuming
+
+With an up-to-date manifest file, a `board_url()` can behave as a read-only version of a `board_folder()`.
+Let's create a `board_url()` using our fake URL:
+
+```{r}
+#| eval: false
+web_board <- board_url("https://not.real.website.co/pins/")
+```
+
+The `board_url()` function reads the manifest file to discover the pins and versions:
+
+```{r}
+#| message: false
+web_board %>% pin_list()
+
+versions <- web_board %>% pin_versions("mtcars") %>% print()
+```
+
+We can read the most-recent version of the `"mtcars"` pin:
+
+```{r}
+#| message: false
+web_board %>% pin_read("mtcars") %>% head()
+```
+
+We can also read the first version:
+
+```{r}
+#| message: false
+web_board %>% pin_read("mtcars", version = versions$version[[1]]) %>% head()
+```
+
+The next section shows some ways you can integrate web-based boards into your workflows.
+
+## Publishing platforms
+
+The goal of this section is to illustrate, for a few platforms, the particular way to publish a board as a part of a website.
+
+### Quarto
+
+Using Quarto, you can [publish](https://quarto.org/docs/publishing/) to GitHub Pages, Quarto Pub, and Netlify, among others.
+
+Let's say you set up a board using a directory called `"pins-board"`; of course, you may wish to use a different name.
+Here's how to set up your Quarto project:
+
+1. In your code, create a `board_folder("pins-board")` in the root directory:
+  - Write to the board using `pin_write()`.
+  - At the end, call `write_board_manifest()`.
+
+2. Add the board folder as a [Quarto site-resource](https://quarto.org/docs/websites/website-tools.html#site-resources):
+
+  - At the top level of your `_quarto.yml` file:
+  
+    ```yml
+    resources:
+      - pins-board
+    ```
+
+  - You may wish to add `/pins-board/` to your project's `.gitignore` file.
+
+Here's an example of a [Quarto repository](https://github.com/ijlyttle/quarto-pins) and its [published website](https://ijlyttle.github.io/quarto-pins/).
+
+### pkgdown
+
+Pins offers another way for R-package developers to share data associated with a package. 
+Publishing a package dataset as a pin can let you extend your data's "audience" to Python users.
```

Comment:
I'm going to take out "Python users" here because Python's `board_url()` doesn't support the manifest yet.

Reply:
In which case, I'll look forward to its return :)